mongo/integration/collection_test.go

```diff
@@ -1886,3 +1910,36 @@ func create16MBDocument(mt *mtest.T) bsoncore.Document {
 	assert.Equal(mt, targetDocSize, len(doc), "expected document length %v, got %v", targetDocSize, len(doc))
 	return doc
 }
+
+func assertGetMoreCommandsAreMonitored(mt *mtest.T, cmdName string, cursorFn func() (*mongo.Cursor, error)) {
```

Comment:
Maybe add a small comment explaining what this and `assertKillCursors...` do? Was not immediately apparent to me that these execute a function and ensure that the following `getMore` or `killCursors` are picked up by `GetStartedEvent()`.

Reply:
Done.